homework/fibonacci/fibonacci.hpp

```diff
@@ -1,11 +1,48 @@
 #pragma once
 
 int fibonacci_iterative(int sequence) {
-    // TODO: Your implementation goes here
+    
+    
+    int tab[sequence+1];
+    tab[0]=0;
+    tab[1]=1;
```

Comment:
Could you tell me which style of programming is it? 

This is nice solution and have special name ;)

Reply:
No, not really. After reading about paradigmats, that might be Imperative style, but this is just a gues.

Reply:
This is dynamic programing. One of most advanced technique :) This is one of the easiest example, but this is really powerful method, for other problems :)